tests/MongoDB.Driver.Core.Tests/Specifications/server-discovery-and-monitoring/tests/rs/compatible_unknown.yml

```diff
@@ -0,0 +1,31 @@
+description: "Replica set member and an unknown server"
+uri: "mongodb://a,b/?replicaSet=rs"
+phases: [
+    {
+        responses: [
+                ["a:27017", {
+                    ok: 1,
+                    ismaster: true,
+                    setName: "rs",
+                    hosts: ["a:27017", "b:27017"],
+                    minWireVersion: 0,
+                    maxWireVersion: 6
+                }],
+        ],
+        outcome: {
+            servers: {
+                "a:27017": {
+                    type: "RSPrimary",
+                    setName: "rs"
+                },
+                "b:27017": {
+                    type: "Unknown",
+                }
+            },
+            topologyType: "ReplicaSetWithPrimary",
+            setName: "rs",
```

Comment:
It looks like we don't check this property itself: https://github.com/mongodb/mongo-csharp-driver/blob/master/tests/MongoDB.Driver.Core.Tests/Specifications/server-discovery-and-monitoring/TestRunner.cs#L166.

But not sure that we should. As I see, this value is related to the values that are set in the `outcome/servers` (line 19) collection and we already have checked them here: https://github.com/mongodb/mongo-csharp-driver/blob/master/tests/MongoDB.Driver.Core.Tests/Specifications/server-discovery-and-monitoring/TestRunner.cs#L237